src/plugins/auto/README.md

```diff
@@ -0,0 +1,15 @@
+# OpenVINO AUTO Plugin
+
+AUTO plugin for OpenVINO™ Runtime, uses a “virtual” or a “proxy” device, which does not bind to a specific type of hardware, but rather selects the processing unit for inference automatically. It detects available devices, picks the best-suited for the task, and configures optimization settings. This way, developer can write the application once and deploy it anywhere. 
```

Comment:
Can you align the document with this guide: https://github.com/openvinotoolkit/openvino/blob/master/docs/dev/dev_doc_guide.md ?